core/src/integration-test/java/com/scalar/db/storage/cassandra/CassandraEnv.java

```diff
@@ -20,13 +20,19 @@ public static Properties getProperties(@SuppressWarnings("unused") String testNa
     String username = System.getProperty(PROP_CASSANDRA_USERNAME, DEFAULT_CASSANDRA_USERNAME);
     String password = System.getProperty(PROP_CASSANDRA_PASSWORD, DEFAULT_CASSANDRA_PASSWORD);
 
-    Properties props = new Properties();
-    props.setProperty(DatabaseConfig.CONTACT_POINTS, contactPoints);
-    props.setProperty(DatabaseConfig.USERNAME, username);
-    props.setProperty(DatabaseConfig.PASSWORD, password);
-    props.setProperty(DatabaseConfig.CROSS_PARTITION_SCAN, "true");
-    props.setProperty(DatabaseConfig.CROSS_PARTITION_SCAN_FILTERING, "false");
-    props.setProperty(DatabaseConfig.CROSS_PARTITION_SCAN_ORDERING, "false");
-    return props;
+    Properties properties = new Properties();
```

Comment:
I'm just wondering why `props` is renamed to `properties`.
It's fine, but other `XXXEnv` don't do the same, I feel it's a bit inconsistent.

Reply:
Yeah, that's true. Let me make things consistent. Thanks.